src/test/java/com/znsio/rpap/steps/AppSteps.java

```diff
@@ -0,0 +1,61 @@
+package com.znsio.rpap.steps;
+
+import com.applitools.eyes.appium.Eyes;
+import com.applitools.eyes.appium.Target;
+import com.epam.reportportal.annotations.Step;
+import com.znsio.rpap.pages.AppExample;
+import io.appium.java_client.AppiumDriver;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.testng.Assert;
+
+import static com.znsio.reportportal.integration.utils.ReportPortalLogger.captureAndAttachScreenshot;
+import static com.znsio.reportportal.integration.utils.ReportPortalLogger.logInfoMessage;
+
+public class AppSteps {
```

Comment:
Why are Steps in test/steps instead of main/steps?

Reply:
As discussed -renamed it to BL in test folder